lib/util.js

```diff
@@ -44,3 +45,18 @@ exports.linkifyCommitRange = (url, commitRange) => {
 
 	return terminalLink(commitRange, `${url}/compare/${commitRange}`);
 };
+
+exports.getTagVersionPrefix = async options => {
```

Comment:
I think we should memoize this call as it never changes during the `np` run. You can just wrap the function here in https://github.com/sindresorhus/p-memoize.

Reply:
I think we should enforce the `yarn` option to be passed here as it's easy to forget it otherwise. Let's use https://github.com/sindresorhus/ow for this. I want to use it other places in the codebase too in the future.